CHANGELOG.asc

```diff
@@ -2,6 +2,31 @@
 [appendix]
 == Release Notes
 
+=== Version 0.2.0 (Release Date: October 11, 2017)
+
+[source, m2]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.2.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Apache Cassandra 2.1.18, 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 0.98.24-hadoop2, 1.2.6
+* Google Bigtable 1.0.0-pre3
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 5.6.2
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
+* Apache TinkerPop 3.2.6
+* Java 1.8
+
+For more information on features and bug fixes in 0.2.0, see these GitHub milestone:
```

Comment:
Looks like I had typo in the 0.1.1 notes that got copied into here: `see these GitHub` -> `see the GitHub`